inc/options/class-wpseo-option-titles.php

```diff
@@ -193,6 +171,29 @@ public function get_separator_options() {
 		return $separators;
 	}
 
+	/**
+	 * Get the available separator human friendly labels.
+	 *
+	 * @return array $friendly_separators Array with the separator human friendly labels.
+	 */
+	public function get_separator_options_for_display() {
+		$separators     = $this->get_separator_options();
+		$separator_list = $this->get_separator_option_list();
+
+		$friendly_separators = array();
+
+		foreach ( $separators as $key => $label ) {
+			$friendly_label = isset( $separator_list[ $key ]['label'] ) ? $separator_list[ $key ]['label'] : '';
+
+			$friendly_separators[ $key ] = array(
+				'label'          => $label,
```

Comment:
So `option` becomes `label` and `label` becomes `friendly_label`.
This is pretty confusing.

Reply:
In part the names come from what was used in the previous implementation and not so ideal to start with: `add_choice( $value, $label, $screen_reader_text = '' )`. Open to suggestions.

Reply:
As the underlaying code uses `screenReaderText` everywhere, I find it more confusing to only change this part to a "friendly"-term which does not imply any use or where you might need to search to find it's use.

I understand that it is not technically correct to use this term, but only changing it in this part does not improve on the way the code can be understood.

Reply:
So I prefer to not change the `sceen_reader_text` in this scope.

Reply:
As this is used as `aria_label` in the `radio` method, would that not be a better name?